scripts/android.mk

```diff
@@ -33,31 +33,24 @@ $(NDK_HOME):
 	# This is put into the root dir, not into $(SRCDIR), because it's huge and
 	# clutters the Travis CI cache.
 	test -f $(NDK_PACKAGE) || curl -s $(NDK_URL) -o $(NDK_PACKAGE)
-	7z x $(NDK_PACKAGE) $(foreach x,$(NDK_FILES),'-ir!$(NDK_DIR)/$x') > /dev/null
-	rm -rf $@
-	mv $(NDK_DIR) $@
+	7z x $(NDK_PACKAGE) -o$(NDK_HOME)_tmp && mv $(NDK_HOME)_tmp/* $(NDK_HOME) && rm -r $(NDK_HOME)_tmp/
```

Comment:
What does this fix? Also, instead of && you should put these on separate lines, since Makefiles are already effectively &&-ing, but that way if one of them fails we'll know which one.